ksvgtopng is being renamed ksvgtopng4 to allow kdebase-runtime be 
co-installable witj KDE 3.
--- a/pics/CMakeLists.txt
+++ b/pics/CMakeLists.txt
@@ -4,9 +4,9 @@ add_subdirectory( emoticons )
 #add_subdirectory( oxygen )
 
 set(ksvgtopng_SRCS ksvgtopng.cpp )
-kde4_add_executable(ksvgtopng NOGUI ${ksvgtopng_SRCS})
-target_link_libraries(ksvgtopng ${KDE4_KDEUI_LIBS})
-install(TARGETS ksvgtopng ${INSTALL_TARGETS_DEFAULT_ARGS} )
+kde4_add_executable(ksvgtopng4 NOGUI ${ksvgtopng_SRCS})
+target_link_libraries(ksvgtopng4 ${KDE4_KDEUI_LIBS})
+install(TARGETS ksvgtopng4 ${INSTALL_TARGETS_DEFAULT_ARGS} )
 
 # create a symlink for the default icon theme,
 # so that fallbacks work even for themes that don't inherit from oxygen
--- a/pics/ksvgtopng.cpp
+++ b/pics/ksvgtopng.cpp
@@ -18,7 +18,7 @@ int main(int argc, char **argv)
 
 	if(argc < 5)
 	{
-		cout << "Usage : ksvgtopng width height svgfilename outputfilename" << endl;
+		cout << "Usage : ksvgtopng4 width height svgfilename outputfilename" << endl;
 		cout << "Please use full path name for svgfilename" << endl;
 		return -1;
 	}
